Displayr/verbs
# DS-3136 Add Sum Feature

Sum function added that computes base::sum for single inputs
and base::`+` for multiple inputs. Also incorporates

* Name matching across row and column dimensions.
  - Using Fuzzy and exact matching
  - Can hide or show unmatched.
* Partial dimension reshaping
* Tidy dimension names for elementwise sums that arent name matched.
* Custom removal of rows and columns by name

﻿- DS-3136 First draft of Sum
- DS-3136 Simplify and improve previous version
- DS-3136 Add first draft of SumRows
- DS-3136 Add first draft of SumColumns
- DS-3136 Add tests for utility functions
- DS-3136 Update warning message to be more flexible
- DS-3136 Update sum row and column tests
- DS-3136 Improve efficiency of opp Inf check
- DS-3136 Add feature to match names
- DS-3136 Minor documentation update
- DS-3136 Update function.name handling
- DS-3136 Refactor checking multiple statistics
- DS-3136 Refactor SumRows
- DS-3136 Modify SumColumns for new behaviour
- DS-3136 Simplify extraction of chart data
- DS-3136 Remove old Inf check function
- DS-3136 Update element matching functions [ci skip]
- DS-3136 Add unit tests for element matching
- DS-3136 Update tests for allowed inputs
- DS-3136 Minor doc update
- DS-3136 Add utils to NAMESPACE
- DS-3136 Add missing argument for function.name
- DS-3136 Remove check of weights in subset code
- DS-3136 Create common Rd file for Sum and SumRows
- DS-3136 Improve tests for Sum
- DS-3136 Placeholder for detailed documentation
- Fix infinity checking and row/col removal
- DS-3136 Update doc for utilities
- DS-3136 Update tests
- fixup! DS-3136 Update tests
- DS-3136 Remove duplicated quoting
- DS-3136 Add sQuote to tests
- DS-3136 Add basic SumRows fuzzy tests
- DS-3136-2 Levenshtein improvements
- DS-3136 Fix mapping error in unmatched fuzzy
- DS-3136 Error early if unmatched observed and cannot be ignored
- DS-3136 Improve SumColumns tests
- DS-3136 Handle fuzzy matching with unmatched out
- DS-3136 Fix unit tests
- DS-3136 Add Rd files of main help file
- DS-3136-2 Consolidate Opposite Inf checks for Sum
- DS-3136 Improve warnings for SumRows
- DS-3136-2 Generalize the check for Opposite Infinities
- Update .travis.yml
- Simplify Opp Infinity check
- DS-3136 Rename subset and weighting function
- DS-3136 Update documentation
- Move test data to testthat dir
- Add logging to travis.yml
- Fix spacing last commit/travis cfg edit
- Add aliases for debugging travis [untracked]
- Remove spaces in alias cmd def in travis.yml
- Restore travis.yml [rev dep skip]
- Add install logging to travis [rev dep skip]
- Fix typos in documentationa [ci skip]
- Start fresh
- DS-3136-3 Update Sum signature
- Create simple exact name matching function
- Improve exact matching tests
- Handle NaN and missing data in Sum
- Add feature to reshape scalars
- Tidy up getDim function
- Improve reshaping
- Add Reduce Sum template and add tests
- Add unmatched handling for exact matching
- Add tests for fuzzy matching [ci skip]
- Add feature to update dimension labels
- Remove SumRows and SumColumns for now
- DS-3136-3 Update Sum examples
- DS-3136-3 Improve warning and error message
- Fix some tests [ci skip]
